app/jobs/auth_trail/geocode_job.rb

```diff
@@ -10,11 +10,11 @@ def perform(login_activity)
         end
 
       if result
-        login_activity.update!(
-          city: result.try(:city).presence,
-          region: result.try(:state).presence,
```

Comment:
It looks like this might change how region is defined.

Reply:
@ankane You're right!  The default result object (`Geocoder::Result::IpinfoIo`) defines `region`, but turns out not all do (e.g. `Geocoder::Result::IpdataCo`).  Plus, it turns out the [docs say](https://github.com/alexreisner/geocoder#custom-result-handling) "Every `Geocoder::Result` object, `result`, provides ... state" (but not necessarily `region`).  Just pushed a commit to address that, which adds a simple mapping.

Reply:
Thanks for updating, but I don’t think an additional mapping is needed. Sorry for the back and forth, but try to keep the change set as minimal as possible from the original version.

Reply:
Okay, changeset now minimized :+1: